### DISCUSSION TITLE
Add Ordinal Indicator Glyphs ( ª º )

### DISCUSSION THREAD

**Discussion by juaogeneroso** [ivodolenc/aspekta#14] [Ideas]
### Request Description

Hello, and thank you for your work!
There's an incredible amount of glyphs, but I found that ordinal indicators ( ª º ) are missing, would it be possible to include them?
Thank you in advance!

### Current Behavior

The issue affects all versions, including the variable and static typefaces.

### Version

2.0.0

**Comment by ivodolenc** [comment]
Hi, thanks for the suggestion. I agree, it could be added.

I will mark this as planned.

Also, since this is a feature request (not a bug), I'll move it to the discussions.

Issues are intended for bug reports only.

**Comment by marlon-borges** [comment]
Hi @ivodolenc, any updates on this?
